content/github/committing-changes-to-your-project/changing-a-commit-message.md

```diff
@@ -102,10 +98,15 @@ If you need to amend the message for multiple commits or an older commit, you ca
   ```
 4. Save and close the commit list file.
 5. In each resulting commit file, type the new commit message, save the file, and close it.
-6. Force-push the amended commits.
-  ```shell
-  $ git push --force
-  ```
+
+{% warning %}
+
+When you're ready to push your changes to GitHub, use the push --force command to force push over the old commit.
+```shell
+$ git push --force <em>example-branch</em>
+``` 
+
+ {% endwarning %} 
```

Comment:
@Ruth-ikegah Instead of having this section in as a warning, can you make it step 6? So, remove the warning tags, add a `6.` to keep it in the ordered list, and this will be good to go!

Reply:
Thanks for reviewing @janiceilene. I have made the changes now.